src/Components/OATConfirmDialog/OATConfirmDialog.tsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import { useOatPageContext } from '../../Models/Context/OatPageContext/OatPageContext';
 import ConfirmDeleteDialog from '../ADT3DSceneBuilder/Internal/ConfirmDeleteDialog/ConfirmDeleteDialog';
 import { OatPageContextActionType } from '../../Models/Context/OatPageContext/OatPageContext.types';
 
 const OATConfirmDialog: React.FC = () => {
     // contexts
     const { oatPageState, oatPageDispatch } = useOatPageContext();
+    const dialogData = oatPageState.confirmDeleteOpen;
```

Comment:
can we rename this confirmDeleteOpen to something else? Because it stores deletion related props in it right?
```suggestion
    const dialogData = oatPageState.confirmDeleteProps;
```

Reply:
Renamed